docs/3.api/3.utils/navigate-to.md

```diff
@@ -81,12 +97,13 @@ await navigateTo('https://nuxt.com', {
 </script>
 ```
 
-### Using open()
+### Opening a Page in a New Tab
 
 ```vue
 <script setup lang="ts">
 // will open 'https://nuxt.com' in a new tab
 await navigateTo('https://nuxt.com', {
+  external: true,
```

Comment:
this isn't needed when used with `open`.

https://github.com/nuxt/nuxt/blob/c01050f7ba931b557ebf66c08331969152942aca/packages/nuxt/src/app/composables/router.ts#L127-L137

```suggestion
```